install_config/install/host_preparation.adoc

```diff
@@ -640,14 +640,11 @@ of log files.
 |Sets the maximum number of log files to be kept per host.
 |===
 
-Edit the `json-file` log driver to use the options you require. When supplying additional options, ensure that you
-maintain the single quotation mark formatting.
-
-The following example indicates that Docker rotates between three 1 MB log files, 
-always keeps the last three log files, and does not require signature verification:
+For example, to set the maximum file size to 1MB and always keep the last three log files, edit the
+`/etc/sysconfig/docker` file to configure `max-size=1M` and `max-file=3`, ensuring that the values maintain the single quotation mark formatting:
 
 ----
-OPTIONS=' --selinux-enabled --log-opt  max-size=1M --log-opt max-file=3 --insecure-registry 172.30.0.0/16 --log-driver=json-file --signature-verification=false'
+OPTIONS= `--insecure-registry=172.30.0.0/16 --selinux-enabled --log-opt  max-size=1M --log-opt max-file=3'
```

Comment:
`--insecure-registry should be '--insecure-registry. (accent used here instead of single quotation mark).